docs-src/references/strongly-typed-errors/cause-unhandled-update.md

```diff
@@ -0,0 +1,15 @@
+---
+id: cause-unhandled-update
+title: Cause Unhandled Update
+description: Explanation for Cause Unhandled Update error message, and how to fix it.
```

Comment:
```suggestion
description: Explanation for Cause Unhandled Update error message, and how to fix it.
```

From the description, this does not seem like something that needs to be fixed, it's just something that happens. Also doesn't seem like an error, but a warning/info? The Update is accepted and applied.